lib/twemoji.rb

```diff
@@ -206,12 +204,20 @@ def self.filter_emojis(content)
     # @return [String] Emoji image tag generated by name and options.
     # @private
     def self.img_tag(name)
-      unicode = render_unicode(name)
-      if options[:img_attr]
-        "<img class='#{options[:class_name]}' draggable='false' title='#{name}' alt='#{unicode}' #{options[:img_attr]} src='#{emoji_url(name)}'>"
-      else
-        "<img class='#{options[:class_name]}' draggable='false' title='#{name}' alt='#{unicode}' src='#{emoji_url(name)}'>"
+      # set default attributes and override from img_attrs
+      attributes = {
+        draggable: "false",
```

Comment:
based on https://github.com/twitter/twemoji i'd leave it in unless there's a reason to take it out